flask/helpers.py

```diff
@@ -931,7 +931,8 @@ def _get_static_url_path(self):
             return self._static_url_path
 
         if self.static_folder is not None:
-            return '/' + os.path.basename(self.static_folder)
+            basename = os.path.basename(self.static_folder)
+            return basename and '/' + basename or ''
```

Comment:
```suggestion
            return '/' + basename if basename else ''
```

Reply:
hah.. i'd been doing mostly shell at the time (hence a command pipeline mental state). agree that that's more pythonic, thus better. would you prefer a forced push or an addition that you squash?